CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Changelog
+## v7.2.1
 
 - rename `toll-booth` to `toll` to reflect `maki_beta` data
```

Comment:
_technically_ this is a breaking change so we might want to actually go `v8` here.

Reply:
even if `7.2.0` wasn't referenced in any project yet? It was released a couple of days ago

Reply:
I see what you are saying - we can't know for sure if someone referenced this icon with the old naming. Updated to `8.0.0`!